src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -61,8 +63,10 @@ public PersonCard(Person person, int displayedIndex) {
         String telegramAddress = person.getTelegramHandle() != null ? " (@" + person.getTelegramHandle().value + ")"
                 : " -";
         name.setText(person.getName().fullName + telegramAddress);
-        attendance.setText(String.format("Attendance: %s",
-                this.person.getStyledAttendanceList("✓", "✕")));
+        attendance.setText(String.format("Attendance: %d/%d  Attendance summary: %s",
+                this.person.getAttendance().getWeeksPresent(), this.person.getAttendance().getTotalMarkedTut(),
+                this.person.getAttendance().getStyledStatusList("❌", "\uD83D\uDE4B\u200D♂\uFE0F",
+                        "\uD83D\uDE47\u200D♂\uFE0F", "?")));
```

Comment:
What are these symbols? Would it be better to leave a comment here explaining what `"\uD83D\uDE4B\u200D♂\uFE0F"` and `"\uD83D\uDE47\u200D♂\uFE0F"` are?

Reply:
Resolved in the updated PR by declaring them as String variables within the PersonCard class.